lib/providers/models/explore_card_filters_model.dart

```diff
@@ -80,14 +86,17 @@ class ExploreCardFiltersModel extends ChangeNotifier {
       maxResultCount: maxResultsCount,
       languagesTextIds:
           _selectedLanguages.isEmpty ? null : _selectedLanguages.toList(),
-      offersHelp: _selectedUserTypes.contains("Mentor")
+      industriesTextIds:
+          _selectedIndustries.isEmpty ? null : _selectedIndustries.toList(),
+      companyStagesTextIds:
+          _selectedStages.isEmpty ? null : _selectedStages.toList(),
+      offersHelp: _selectedUserType == UserType.mentor
           ? Enum$UserSearchFieldPreference.isTrue
-          : Enum$UserSearchFieldPreference.any,
+          : Enum$UserSearchFieldPreference.isFalse,
```

Comment:
We do have some users in our current app who are both mentors and mentees. With this logic, we'd never show those users, which is why I'd asked Michal to use the `any` option as a backup. 

Not a big deal for the moment, since we haven't designed any other features for users who have both entrepreneur and mentor roles yet. At some point we're likely to change this though